code/game/objects/machinery/squad_supply/supply_packer.dm

```diff
@@ -0,0 +1,26 @@
+/obj/structure/largecrate/packed
+	name = "supplies crate"
+	icon_state = "secure_crate"
+	var/list/manifest = list("This crate contains:")
+	var/static/list/black_list = typecacheof(list(/obj/structure))
+	var/max_items = 30 // while packing the entire FOB into a single crate would be funny propably not great idea
+
+/obj/structure/largecrate/packed/Initialize()
+	. = ..()
+	pack()
+
+/obj/structure/largecrate/packed/examine()
+	. = ..()
+	for(var/item in manifest)
+		. += span_boldnotice(item)
+
+/obj/structure/largecrate/packed/proc/pack()
+	var/count = 0
+	for(var/obj/thing in loc)
```

Comment:
Should probably be obj/item instead of just obj so you don't try to pack up grilles or sticky resin.

Reply:
ye but maby figured you'd want to back like motorcycles or other such stuff. but ill adjust it